webpack.config.js

```diff
@@ -109,23 +109,29 @@ module.exports = (env = {}) => {
     module: {
       rules: [
         {
-          test: /\.js$/,
+          test: /\.(ts|js)x?$/,
           exclude: /node_modules/,
           use: [
             {
               loader: 'babel-loader',
               options: {
                 presets: [
                   [
-                    'env',
+                    '@babel/preset-env',
                     {
                       modules: false,
-                      exclude: ['transform-es2015-function-name'],
+                      targets: {
+                        browsers: ['last 1 version', 'ie >= 11'],
+                      },
```

Comment:
I think the defaults are usually fine, this seems like a small target. But sounds like specifying nothing isn't the same as specifying defaults with babel https://babeljs.io/docs/en/babel-preset-env#no-targets.

So this?
```suggestion
                      targets: 'defaults',
```

`defaults` is `> 0.5%, last 2 versions, Firefox ESR, not dead` according to https://github.com/browserslist/browserslist#full-list